core/server/data/schema/versioning.js

```diff
@@ -1,78 +0,0 @@
-var Promise = require('bluebird'),
-    db = require('../db'),
-    errors = require('../../errors'),
-    ghostVersion = require('../../utils/ghost-version');
-
-/**
- * Database version has always two digits
- * Database version is Ghost Version X.X
- *
- * @TODO: remove alpha text!
- * @TODO: extend database validation
- */
-function validateDatabaseVersion(version) {
-    if (version === null) {
-        throw new errors.DatabaseVersionError({
-            code: 'VERSION_DOES_NOT_EXIST'
-        });
-    }
-
-    if (!version.match(/\d\.\d/gi)) {
```

Comment:
The compatibility check moved into `core/server/index.js`.
https://github.com/TryGhost/Ghost/pull/7894/files#diff-79a58308a1a36ab1b54544a17f88997aR87